test/getUriParameters-test.coffee

```diff
@@ -116,7 +116,7 @@ describe('Transformation • Refract • getUriParameters' , ->
           }
         ]
       },
-      # Parameter is required by default with no type attributes
+      # Parameter is optional by default with no type attributes
```

Comment:
I think it would be great to add a test coverage for each scenario. Required parameter, optional parameter and a parameter with no type attributes.

Could you add two more parameters to the above list and change this section on parameter required/optional/behaviour?